modules/swagger-codegen/src/main/resources/python/tornado/rest.mustache

```diff
@@ -41,30 +39,10 @@ class RESTClientObject(object):
 
     def __init__(self, configuration, pools_size=4, maxsize=4):
         # maxsize is number of requests to host that are allowed in parallel
-        # ca_certs vs cert_file vs key_file
-        # http://stackoverflow.com/a/23957365/2985775
 
-        # ca_certs
-        if configuration.ssl_ca_cert:
-            ca_certs = configuration.ssl_ca_cert
-        else:
-            # if not set certificate file, use Mozilla's root certificates.
-            ca_certs = certifi.where()
```

Comment:
@tomplus shall we keep the default certificate `certifi.where()`?

Reply:
This is also handled by the tornado client it seems based on the link provided.

Reply:
Latest versions of Tornado use `certifi`. But it's worth to mention that unstable version from `master` doesn't have this dependency, certificates from system will be used. A developer will still be able to add `certifiy` if she/he really needs to do it.